src/kernel/task/resource/atexit/atexit.h

```diff
@@ -14,6 +14,7 @@
 
 struct task_atexit {
 	void (*atexit_func[ATEXIT_MAX])(void);
+	size_t slot;
```

Comment:
i think that it's better to use simple types if there is not necessary in the opposite, i think it's enough to use 'int' there

Reply:
done